src/thor/alternates.cc

```diff
@@ -15,7 +15,13 @@ using namespace valhalla::midgard;
  */
 namespace {
 // Defaults thresholds
+// TODO: global stretch parameter should depend on the optimal route cost/duration.
```

Comment:
I'm sure that we miss some good alternatives for short routes because of one global threshold. Added as "TODO", but it should be transformed into an issue/PR

Reply:
and, imho, we should also refactor `max_sharing` coefficient in the following way:
1. a little bit increase `max_sharing` coefficient for shorter routes. I think value `0.6` for routes < 20km is too strict, it should be ~0.7.
2. replace piecewise constant function with a smoother one (cubic?) to ensure that for two close routes `max_sharing` constant values also will be close. It's a little bit strange that for 9.9-km route we will use `max_sharing` constant equals to `0.5` but for 10-km route this constant will be `0.6`.

Reply:
both of these suggestions make way more sense to me as well :+1: